### PR TITLE
Clarify support for start levels

### PR DESCRIPTION
**Pull request by kwin**
Document how run mode and start level is determined

### CODE REVIEW

**Comment by rombert** [comment]
@DominikSuess / @niekraaijmakers - does this look good to you?

**Comment by kwin** [comment]
Somehow the detected start levels never end up in the generated feature model. Instead always the default (i.e. the one passed to `--bundles-start-order` is being used for all bundles extracted from packages. Don't know yet if bug or feature...

File: README.md [hunk 1]
```diff
@@ -566,7 +573,7 @@ Apache Sling Content Package to Sling Feature converter
                             The output directory where the artifacts will be
                               deployed.
   -b, --bundles-start-order=<bundlesStartOrder>
-                            The order to start detected bundles.
+                            The default start level in which to start detected bundles.
```
Comment (kwin) [inline README.md:576]:
```suggestion
                            The default start order in which to start detected bundles.
```

**Comment by niekraaijmakers** [comment]
Looks good to me. Assuming the regex is what i think it is (correct) :)

**Pull request merged**